src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_b_fs_yx_fsv16_1x1.cpp

```diff
@@ -307,8 +307,13 @@ JitConstants ConvolutionKernel_b_fs_yx_fsv16_1x1::GetJitConstants(const convolut
         jit.AddConstant(MakeJitConstant("OUTPUT_LEFTOVERS", output_leftover));
 
         const auto input_leftover_num = "(" + input0_dims.f() + "%" + feature_block_size + ")";
-        const auto input_leftover = "(" + input_leftover_num + "!= 0)";
-        jit.AddConstant(MakeJitConstant("INPUT_LEFTOVERS", input_leftover));
+        if (!params.inputs[0].Feature().is_dynamic) {
```

Comment:
Frankly, I don't understand why added to check whether dynamic or static?
As I know, we can check whether dynamic or not in Line 237 and SA kernel can run leftover or not depends on feature size from input param in runtime.

Reply:
Updated to fix in cl kernel code side not here. Thanks!